frontend/src/app/modules/admin/types/type-form-configuration.component.ts

```diff
@@ -30,8 +29,10 @@ export interface TypeGroup {
   type:TypeGroupType;
 }
 
+export const adminTypeFormConfigurationSelector = 'admin-type-form-configuration';
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/3b1eae5a64159f8a044579a25870c772186dbd3f/frontend/src/app/modules/admin/types/type-form-configuration.component.ts#L33"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>